packages/website/pages/pricing.js

```diff
@@ -1,34 +1,405 @@
-import { useEffect } from 'react';
+import Img from 'components/cloudflareImage.js';
+import GradientBackground from 'components/gradientbackground/gradientbackground.js';
+import Scroll2Top from 'components/scroll2top/scroll2top.js';
+import ImageCross from 'public/images/illustrations/cross.png';
+import ImageCoil from 'public/images/illustrations/coil.png';
+import ImageCorkscrew from 'public/images/illustrations/corkscrew.png';
+import ImageTriangle from 'public/images/illustrations/triangle.png';
+import ImageTriangle1 from 'public/images/illustrations/triangle1.png';
+import BlobCluster from 'public/images/illustrations/blob-cluster.png';
+import Cluster1 from 'public/images/index/cluster-1.png';
+import Button from 'ZeroComponents/button/button';
 
-import TiersPageData from '../content/pages/pricing.json';
-import Scroll2Top from '../components/scroll2top/scroll2top.js';
-import BlockBuilder from '../components/blockbuilder/blockbuilder.js';
-import { initFloaterAnimations } from '../lib/floater-animations.js';
+const PricingHeader = () => (
+  <>
+    <section id="section_pricing_header" className="sectional">
+      <div className="grid-middle">
+        <div className="col-10_sm-8_mi-10_ti-12 column-1" data-push-left="off-0_sm-2_mi-1_ti-0">
+          <div className="column-content">
+            <div id="intro_1-hero-corkscrew">
+              <Img alt="" src={ImageCorkscrew} />
+            </div>
+            <div id="intro_1-coil">
+              <Img alt="" src={ImageCoil} />
+            </div>
+            <div id="intro_1-cross">
+              <Img alt="" src={ImageCross} />
+            </div>
+            <div className="block text-block format__header">
+              <h1 className="h1 heading">Storage that grows with you</h1>
+              <div className="description">
+                <p>
+                  Web3.Storage is designed for scale and simplicity. Utilize our elastic, hosted data platform that
+                  natively integrates decentralized data and authentication protocols. No need to worry about
+                  performance or reliability.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="section_trusted_by_header" className="sectional">
+      <div className="grid-middle">
+        <div className="col-10_sm-8_mi-10_ti-12 column-1" data-push-left="off-0_sm-2_mi-1_ti-0">
+          <div className="column-content">
+            <div className="block text-block format__medium">
+              <div className="subheading">TRUSTED BY</div>
```

Comment:
I love to use `text-transform: uppercase` css for text like this, that way different stylesheets can style things differently. https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

more just an fyi but not a must-change

Reply:
The only place I've ever been really comfortable with `text-transform: uppercase` is with non-static data.  When you can control the input string, I find `TRUSTED BY` (the same way it'll render) as being more intuitive than `trusted by` with the transform.  It's probably splitting hairs / opinion zone though.  So if you'd prefer text-transform I'm happy to do it.